types/mux.d.ts

```diff
@@ -141,7 +150,17 @@ export declare class LiveStreams extends Base {
 export declare class Uploads extends Base {
   create(params: CreateUploadParams): Promise<Upload>;
   cancel(uploadId: string): Promise<Upload>;
-  get(uploadId: string): Promise<Upload>;
```

Comment:
@dylanjha Is there a reason this was removed? It seems like this function still exists.

Reply:
@kesne yikes. It looks like that was an oversight. The API definitely still exists.

Reply:
fixed in [3.1.3](https://github.com/muxinc/mux-node-sdk/releases/tag/v3.1.3), thank you!